### ISSUE TITLE
Specify spare pieces

### ISSUE THREAD

**Issue by Tjorriemorrie** [oakmac/chessboardjs#115]
Just to confirm: is it not possible to specify spare pieces? It seems you can only pass boolean parameter.

**Comment by oakmac** [comment]
Correct; you cannot specify which pieces show up as spare currently.

**Comment by Tjorriemorrie** [comment]
I would like to add that feature; any advice on where to begin or the section of code is?

**Comment by user14159** [comment]
In chessboard-0.3.0.js look under the heading Markup Building for the function buildSparePieces. 

```
function buildSparePieces(color) {
  var pieces = ['wK', 'wQ', 'wR', 'wB', 'wN', 'wP'];
  if (color === 'black') {
    pieces = ['bK', 'bQ', 'bR', 'bB', 'bN', 'bP'];
  } ...

```
Delete whichever pieces you don't want. For example, remove **'wR',** to remove the white rook.
